app/lib/tariff_synchronizer.rb

```diff
@@ -309,7 +309,7 @@ def sequel_models
   end
 
   def check_sequence_for_pending_file
-    unless update_type.correct_filename_sequence?
+    unless update_type.correct_recent_filename_sequence?
```

Comment:
this could be a bit confusing, it's not easy to say the difference between
`correct_filename_sequence?`
`correct_recent_filename_sequence?`

should be rename `correct_filename_sequence?` to something like `compare_pair_sequence?`
and 
`correct_recent_filename_sequence?` to just `correct_filename_sequence`?